sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/BarrierRequestHelper.java

```diff
@@ -115,10 +115,12 @@
                 break;
 
             default:
-                String unknownAuthToken = "Unknown authorization token kind for read request";
+                String unknownAuthToken =
+                    "Unknown authorization token kind '" + originalRequestTokenType + "' for read request";
```

Comment:
What are the other possible kinds? (SAS is not yet ready)

Reply:
No clue - that is why I added the logs :-) Customer in CRI is claiming they use master key